### PR TITLE
Merge racon-gpu changes upstream

### PR DESCRIPTION
**Pull request by tijyojwad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ To install Racon run the following commands:
 
 ```bash
 git clone --recursive https://github.com/lbcb-sci/racon.git racon
-cd racon
+cd racon-gpu
```
Comment (rvaser) [inline README.md]: Was this change included by accident?

Best regards,
Robert

**Comment by tijyojwad** [comment]
@rvaser can you have a look at this PR when possible?

**Comment by rvaser** [comment]
I am sorry @tijyojwad, I wrote one comment the day you opened the pull request, but did not start the review.

**Comment by tijyojwad** [comment]
@rvaser no worries! That feature of the GitHub UI has tripped me up several times too. Fixed up the README and updated version number as well now

**Pull request merged**